sapphire/analysis/calibration.py

```diff
@@ -86,6 +93,196 @@ def determine_detector_timing_offset(dt, dz=0):
     return detector_offset, rchi2
 
 
+class DetermineStationTimingOffsets(object):
+    """Determine the timing offsets between stations"""
+
+    MAX_DISTANCE = 1000  # m
+    """maximum distance between station pairs that are included in analysis"""
+    MIN_LEN_DT = 100
+    """minimum number of timedeltas required to attempt a fit"""
+
+    def __init__(self, stations=None, data=None, progress=False,
+                 force_stale=False):
+        """ Initialize the class
+
+        :param data: pytables HDF5 file with timedelta tables
+        :param stations: list of station for which to determine station offsets
+        :param progress: if true: show progressbar if true
+        :param force_stale: if true: do not get network information from API.
+
+        """
+        self.data = data
+        self.progress = progress
+        self.force_stale = force_stale
+        if stations is not None:
+            self.cluster = HiSPARCStations(stations, skip_missing=True,
+                                           force_stale=self.force_stale)
+        else:
+            self.cluster = HiSPARCNetwork(force_stale=self.force_stale)
+
+    def read_dt(self, station, ref_station, start, end):
+        """ Overload this method on publicdb """
+
+        table = self.data.get_node('/s%d' % station)
```

Comment:
I changed the path. (This was the path in my testfile, it matches the one in the ProcessTimeDeltas class now).
```read_dt()``` is overloaded on publicdb, but it uses the same path